### ISSUE TITLE
NPEs on starting up inspect

### ISSUE THREAD

**Issue by ghoustonjr** [matthiasn/inspect#7]
Just wanted to let you know, there appears to a problem with the jar at clojars for version 0.1.4 of inspect.

I brought inspect into my project with the following:

``` clojure
[com.matthiasnehlsen/inspect "0.1.4"]
``` 

I started inspect by executing com.matthiasnehlsen.inspect/start, then visited http://localhost:8080 and saw the following logged: 
```
2015-Jan-19 23:08:03 -0500 Home-PC ERROR [taoensso.sente] - Chsk router handling error: [:cmd/get-stats]
java.lang.NullPointerException: 
                                               clojure.core/subs                 core.clj: 4765
                   com.matthiasnehlsen.inspect.matcher/strip-uid              matcher.clj:   25
       com.matthiasnehlsen.inspect.matcher/make-handler/fn/fn/fn              matcher.clj:   63
          com.matthiasnehlsen.inspect.matcher/make-handler/fn/fn              matcher.clj:   60
             com.matthiasnehlsen.inspect.matcher/make-handler/fn              matcher.clj:   60
taoensso.sente/start-chsk-router!/fn/state-machine--auto--/fn/fn                sente.clj: 1084
   taoensso.sente/start-chsk-router!/fn/state-machine--auto--/fn                sente.clj: 1078
      taoensso.sente/start-chsk-router!/fn/state-machine--auto--                sente.clj: 1071
            clojure.core.async.impl.ioc-macros/run-state-machine           ioc_macros.clj:  940
    clojure.core.async.impl.ioc-macros/run-state-machine-wrapped           ioc_macros.clj:  944
                                 clojure.core.async/ioc-alts!/fn                async.clj:  362
                                clojure.core.async/do-alts/fn/fn                async.clj:  231
           clojure.core.async.impl.channels.ManyToManyChannel/fn             channels.clj:  102
                                            clojure.lang.AFn.run                 AFn.java:   22
               java.util.concurrent.ThreadPoolExecutor.runWorker  ThreadPoolExecutor.java: 1142
              java.util.concurrent.ThreadPoolExecutor$Worker.run  ThreadPoolExecutor.java:  617
                                            java.lang.Thread.run              Thread.java:  745
```
The page loads, but no events are logged. Any interaction with the UI generates NPEs like the one above, although the specifics differ a bit. For example:

```
2015-Jan-19 23:08:03 -0500 Greg-PC ERROR [taoensso.sente] - Chsk router handling error: [:cmd/initialize {:n 10}]
java.lang.NullPointerException: 
                                               clojure.core/subs                 core.clj: 4765
                   com.matthiasnehlsen.inspect.matcher/strip-uid              matcher.clj:   25
        com.matthiasnehlsen.inspect.matcher/initialize-inspector              matcher.clj:   35
          com.matthiasnehlsen.inspect.matcher/make-handler/fn/fn              matcher.clj:   62
             com.matthiasnehlsen.inspect.matcher/make-handler/fn              matcher.clj:   60
taoensso.sente/start-chsk-router!/fn/state-machine--auto--/fn/fn                sente.clj: 1084
   taoensso.sente/start-chsk-router!/fn/state-machine--auto--/fn                sente.clj: 1078
      taoensso.sente/start-chsk-router!/fn/state-machine--auto--                sente.clj: 1071
            clojure.core.async.impl.ioc-macros/run-state-machine           ioc_macros.clj:  940
    clojure.core.async.impl.ioc-macros/run-state-machine-wrapped           ioc_macros.clj:  944
                                 clojure.core.async/ioc-alts!/fn                async.clj:  362
                                clojure.core.async/do-alts/fn/fn                async.clj:  231
           clojure.core.async.impl.channels.ManyToManyChannel/fn             channels.clj:  102
                                            clojure.lang.AFn.run                 AFn.java:   22
               java.util.concurrent.ThreadPoolExecutor.runWorker  ThreadPoolExecutor.java: 1142
              java.util.concurrent.ThreadPoolExecutor$Worker.run  ThreadPoolExecutor.java:  617
                                            java.lang.Thread.run              Thread.java:  745
```

Just to rule out an issue with my project, I created a brand new project and brought in v. 0.1.4 of inspect. I had the same problems.

Cloning the project, building, and installing locally (just with 'lein install') is working very well. Unless I'm mistaken, what I've built locally should be the same as what's currently deployed to clojars, so I'm poking around to see if perhaps something went wrong with the build before. Just wanted to let you know about this in case you see an obvious solution.

Thanks for inspect!

**Comment by matthiasn** [comment]
Thank you for the report, I will try to check one of the next days in a fresh project with the dependency pulled in from Clojars. I did some work on it with some newer versions of dependencies recently, didn't get around to publishing it yet though.

**Comment by matthiasn** [comment]
Hi, I finally got around to this and published a new version: v0.1.5
Please check if this works and if not, feel free to reopen the issue. Thanks again!
